contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -17,13 +17,54 @@ def wait_for_main_window(application: Application):
     wait_for_condition(lambda: application.top_window().class_name() == "OrbitMainWindow", 30)
 
 
+def wait_for_connection_window(application: Application):
+    wait_for_condition(lambda: application.top_window().class_name() == "orbit_session_setup::SessionSetupDialog", 30)
+
+
 def _get_number_of_instances_in_list(test_case: E2ETestCase) -> int:
     instance_list = test_case.find_control('Table', 'InstanceList')
     instance_count = instance_list.item_count()
     logging.info('Found %s rows in the instance list', instance_count)
     return instance_count
 
 
+class LoadCapture(E2ETestCase):
+    """
+    Opens a given capture file. The argument `capture_file_path` specifies the full path to the capture file.
+    It raises an exception if the capture does not exists. Also waits until the capture is loaded.
+    """
+
+    def _execute(self, capture_file_path: str):
+        # Lets ensure that we are actually in the connection window, as this test might also be executed after the
+        # main window has been opened.
+        wait_for_connection_window(self.suite.application)
+        self.suite.top_window(force_update=True)
+        logging.info('Start loading to a capture.')
```

Comment:
This string is broken: "Starting to load a capture"?

Reply:
Thanks